File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/non-plagiarized/08/T03.kt
package IR

import java.util.Scanner

/**
 * @author 6ED61D4B80BB0F81937B32418E98ADCA A94652AA97C7211BA8954DD15A3CF838
 */
fun main() {
    val sc = Scanner(System.`in`)
    print("Enter weight in pounds: ")
    val weight = sc.nextDouble()
    print("Enter feet: ")
    val feet = sc.nextDouble()
    print("Enter Inches: ")
    val inches = sc.nextDouble()
    val height = feet * 12 + inches
    val bmi = weight * 0.45359237 / Math.pow((height * 0.0254), 2.0)
    println("BMI is $bmi")
    when {
        bmi < 18.5 -> println("underweight")
        bmi < 25 -> println("normal")
        bmi < 35 -> println("overweight")
        else -> println("obese")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/non-plagiarized/12/T03.py
import math

print("Enter weight in pounds : ", end="")
weight = float(input())

print("Enter feet : ", end="")
feet = float(input())

print("Enter inches : ", end="")
inches = float(input())

height = feet * 12 + inches
bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))
print("BMI is", bmi)

if bmi < 18.5:
    print("Underweight")
elif bmi >= 18.5 and bmi < 25:
    print("Normal")
elif bmi >= 25 and bmi < 35:
    print("Overweight")
else:
    print("obese")

